gwpy/table/table.py

```diff
@@ -304,7 +304,7 @@ def fetch(cls, format_, *args, **kwargs):
 
         To download a table of all blip glitches from the Gravity Spy database:
 
-        >>> EventTable.fetch('gravityspy', 'glitches', selection='Label=Blip')
+        >>> EventTable.fetch('gravityspy', 'glitches', selection=['ml_label=Blip', 'ml_confidence>0.9'])
```

Comment:
This line's too long, perhaps line wrap one argument per line.

```suggestion
        >>> EventTable.fetch(
        ...     'gravityspy',
        ...     'glitches',
        ...     selection=['ml_label=Blip', 'ml_confidence>0.9'],
        ... )
```